www/js/intro.js

```diff
@@ -58,6 +58,10 @@ angular.module('emission.intro', ['emission.splash.startprefs',
       }
     });
   };
+  $scope.startSurvey = function () {
+      SurveyLaunch.startSurvey('https://docs.google.com/forms/d/e/1FAIpQLSd47sf_0bieu81-HtXO5PV3EmOdUfDjWE5xrLSzQ-1hVrgLgQ/viewform?usp=sf_link', 'QR~QID3');
+      startSurvey();
```

Comment:
what is this `startSurvey()` for?

Reply:
This is for launching the Initial Survey after they log in.

Reply:
Yeah but isn't the `SurveyLaunch.startSurvey` enough for that?
why do you need a separate `startSurvey`?

Reply:
Oh huh, I think I was going to add extra functionality to that method but moved it to the username one. 

Will change.